contracts/mocks/GovernorVoteMock.sol

```diff
@@ -3,10 +3,10 @@
 pragma solidity ^0.8.0;
 
 import "../governance/extensions/GovernorCountingSimple.sol";
-import "../governance/extensions/GovernorVotesERC721.sol";
+import "../governance/extensions/GovernorVotes.sol";
 
-contract GovernorERC721Mock is GovernorVotesERC721, GovernorCountingSimple {
-    constructor(string memory name_, ERC721Votes token_) Governor(name_) GovernorVotesERC721(token_) {}
+contract GovernorVoteMock is GovernorVotes, GovernorCountingSimple {
```

Comment:
Please change `GovernorVoteMock` to `GovernorVotesMocks` (with an `s`). Just trying to keep naming convention consistent.